examples/pom.xml

```diff
@@ -70,6 +71,8 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <testcontainers.group>com.github.testcontainers.testcontainers-java</testcontainers.group>
         <testcontainers.version>-SNAPSHOT</testcontainers.version>
+        <cucumber.vesrion>2.3.1</cucumber.vesrion>
```

Comment:
no need to put it here, it is not shared between all examples

Reply:
updated